editor/src/clj/editor/scene_selection.clj

```diff
@@ -133,8 +136,12 @@
                            (g/set-property self :current cursor-pos)
                            (g/set-property self :mode mode)
                            (g/set-property self :toggle? toggle?)
+                           (g/set-property self :contextual? contextual?)
                            (g/set-property self :prev-selection (g/node-value self :selection))))
                        (select self op-seq mode toggle?)
+                       (when contextual?
+                         (-> (ui/init-context-menu! ::scene-selection-menu (.getScene (:parent action)))
+                             (.show (:parent action) (:screen-x action) (:screen-y action))))
```

Comment:
Simply registering a context menu wouldn't work in this case properly. We need to explicitly show it on certain conditions, and make it work properly along with element selection.